services/tasks-api/migrations/2020-04-07T12-55.events.sql

```diff
@@ -0,0 +1,6 @@
+--events (up)
```

Comment:
There's a extra table which keeps track of the names of the migrations that have been ingested and the hash of the file. On boot it will compare and run missing ones (using the down scripts to rollback if there's a problem).

https://github.com/mmkal/slonik-tools/tree/master/packages/migrator#readme

Simple, all hand-written SQL with small automation for running that process above.